nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-test/src/test/resources/csv-always-loaded-directory-contrib.override.withAutoincrementId.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<!--
+  ~  (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
```

Comment:
We don't put copyright headers in XML files (especially for tests), it's overkill.